### DISCUSSION TITLE
Люди нашего комьюнити

### DISCUSSION THREAD

**Discussion by hola-asya** [tech-connect-meetups/tech_vlc#10] [Show and tell]
После одного из мероприятий нам поступила идея сделать публичную "адресную книгу" участников комьюнити.

Это можно сделать здесь! 💃

 **Правила:**
1. Одна ветка - один человек
2. Быть милашками

Структура поста
```
## **Имя:**
Фотка <img src="INSERT URL" width="200">
**Специализация:**
**Супер-сила:** 
**Соц сети:**
**Открыт(а) к:**

```

**Comment by hola-asya** [comment]
## Ася Исакова
<img src="https://optim.tildacdn.com/tild3461-3861-4839-b836-353034316435/-/resize/443x/-/format/webp/IMG_6197.JPG" width="200">

**Специализация:** Организационный психолог, консультант для лидов, ко-фаундер этого сообщества
**Супер-сила:** Создавать условия, в которых люди расцветают (даже на удаленке!)
**Соц сети:** [LinkedIn](https://www.linkedin.com/in/asyaisakova/), [ТГ-канал с полезностями для руководителей](https://t.me/integritycompass)
**Открыт(а) к:** приключениям) готова делать доклады, деврелить, менторить руководителей и помогать строить команды мечты

**Comment by katemalinskaia** [comment]
## **Катерина Малинская**
<img src="https://github.com/tech-connect-meetups/tech_vlc/assets/167994627/60193a87-6285-426c-ad46-be7bbb88797c" width="200">

**Специализация:** Карьерный коуч, консультант по развитию карьеры
**Супер-сила:** Сокращаю время поиска себя и экономлю годы жизни
**Соц сети:** [Instagram](https://instagram.com/kate.mal), [LinkedIn](https://www.linkedin.com/in/katemal/)
**Открыт(а) к:** Общению и партнерствам) Готова организовывать классные мероприятия, консультировать специалистов и помогать вам "перепридумывать" себя и свою карьеру

**Comment by Alexmalinskiy** [comment]
### Алексей Малинский

![IMG_1035___ (5)](https://github.com/tech-connect-meetups/tech_vlc/assets/29104642/e170546b-3fa1-48db-8929-cfc68bc13c41)


**Специализация**: Продуктовый аналитик, руковожу командой 20+ человек
**Супер-сила**: Понимаю в числах, коммуникациях с людьми, выстраивании процессов в IT компаниях
**Соц сети:** Веду свой [канальчик](https://t.me/lexmalinskiy) в ТГ
**Открыт(а) к**: Совместным мероприятиям, разговорам на профессиональные темы и не очень

**Comment by GoSvetlanada** [comment]
## **Света Данильченко**
<img width="200" alt="Снимок экрана 2024-04-26 в 17 17 00" src="https://github.com/tech-connect-meetups/tech_vlc/assets/163444556/4e3074ce-75ef-4fef-ad31-0baf4a9b88a0">

**Специализация:** Tech-бренд и DevRel-менеджер, копирайтер
**Суперсила:** Объединяю людей 
**Соц сети:** Веду телегу [о жизни](https://t.me/Vaaaamos ), о [мероприятиях в Валенсии](https://t.me/gosvetlanada), [Instagram](https://www.instagram.com/svetlanada) 
**Открыта к:** айтишному-движу

**Comment by Kotkoa** [comment]
## **Андрей Котко:**
<img src="https://i.ibb.co/47VWN6q/IMG-2181.jpg" width="200">

**Специализация:** Фронтенд разработчкик - React, Typescrypt, GraphQL
**Супер-сила:** Живу на границе регионов Валенсия и Аликанте в Pego. Люблю делать сайты из красивого UI. Внимателен к деталям. Ещё я нейроотличный человек
**Соц сети:** [Linkedin](https://www.linkedin.com/in/kotkoa/), [Twitter](https://twitter.com/kotkoa/)
**Открыт к:** Я за любой движ. Открыт к предложениям по работе

**Comment by YanaParsh** [comment]
## **Паршина Яна:**
<img src=https://github.com/user-attachments/assets/e41f7f0d-7d22-4857-a0d6-fa59739a5ba6 width="200">

**Специализация:** Product Manager
**Супер-сила:** Вдохновлять на развитие и профессиональный рост, улучшение процессов внутри ИТ-команд и управление продуктами через ориентацию на пользователя
**Соц сети:** [LinkedIn](https://www.linkedin.com/in/iana-parshina/), [parshina.pro](https://parshina.pro)
**Открыта к:** Организации совместных мероприятий, выступлениям, развитию профессионального сообщества, менторству
